kernel/bpf/bpf_io_filter.c

```diff
@@ -146,6 +146,18 @@ static bool io_filter_is_valid_access(int off, int size,
 	return true;
 }
 
+static int io_filter_bpf_attach(struct gendisk *disk, struct bpf_prog *prog)
```

Comment:
I believe attach/detach can happen concurrently with IOs to the disk. May need some locks here

Reply:
Added rcu locking